lnbits/db.py

```diff
@@ -179,16 +179,27 @@ async def fetch_page(
         values: Optional[List[str]] = None,
         filters: Optional[Filters] = None,
         model: Optional[Type[TRowModel]] = None,
+        group_by: Optional[List[str]] = None,
```

Comment:
The users need to know that they have to add an aggregator function (`max`, `count`, etc) when using `group_by`. Otherwise the query execution will fail.

Can this requirement be documented somewhere?